Utilities/Services/PostDeserializer.cs

```diff
@@ -23,12 +23,15 @@ public PostDeserializer(IXMLFileValidator fVal)
             fileValidator = fVal;
         }
 
-        public PostDeserializer(IXMLFileValidator fVal, string dFile, string dXSD)
+        public PostDeserializer(IXMLFileValidator fVal, string dFile, string dXSD, bool loadCacheOnCreation=true)
```

Comment:
I think you're still making the deserializer more complicated than it needs to be. Just have it load on constructor and return that data on request - there's no reason to store the filename or xsd name, all of that should be dealt with by the time the constructor is finishes, and it's just unnecessary complexity to have a boolean that changes its behavior.

(Honestly, it's not even really a deserializer at this point, it's a provider)